app/base_api_error.py

```diff
@@ -7,5 +7,5 @@ def __init__(self, message, error_code):
         self.message = message
         self.error_code = error_code
 
-    def buildError(self):
-        return jsonify({"success": False, "message": self.message}), self.error_code
+    def build_error(code, message):
+        return BaseApiError(message, code)

```

Comment:
I think we're sticking with how it was so this change can be removed. Actually could can keep the `build_error` if you update it everywhere else.